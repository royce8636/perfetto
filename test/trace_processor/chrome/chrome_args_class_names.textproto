packet {
  timestamp: 0
  incremental_state_cleared: true
  trusted_packet_sequence_id: 1
  track_descriptor {
    uuid: 12345
    thread {
      pid: 123
      tid: 345
    }
    parent_uuid: 0
    chrome_thread {
      thread_type: THREAD_POOL_FG_WORKER
    }
  }
}

packet {
  trusted_packet_sequence_id: 1
  timestamp: 0
  incremental_state_cleared: true
  track_event {
    track_uuid: 12345
    categories: "cat1"
    type: 3
    name: "name1"
    [perfetto.protos.ChromeTrackEvent.android_view_dump] {
      activity {
        name: "A"
        view {
          class_name: "abc"
        },
        view {
          class_name: "def"
        },
        view {
          class_name: "ghi"
        }
      }
      activity {
        name: "B"
        view {
          class_name: "jkl"
        }
      }
    }
  }
}
