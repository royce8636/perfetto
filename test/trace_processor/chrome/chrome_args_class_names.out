
[perfetto.protos.chrome_args_class_names] {
  class_names_per_version {
    class_name: "abc"
    class_name: "def"
    class_name: "ghi"
    class_name: "jkl"
  }
}
